src/igvfd/searches/configs/rodent_donor.py

```diff
@@ -38,6 +38,7 @@ def rodent_donor():
                 'facet_fields': [
                     'strain_background',
                     'sex',
+                    'virtual',
```

Comment:
Collection is missing an 's' in L47, not sure if I missed or is from another ticket

Reply:
added s